cruise-control-core/src/test/java/com/linkedin/cruisecontrol/CruiseControlUnitTestUtils.java

```diff
@@ -16,6 +16,7 @@
   public static final String METRIC1 = "m1";
   public static final String METRIC2 = "m2";
   public static final String METRIC3 = "m3";
+  public static final String CPU_USAGE = "CPU_USAGE";
```

Comment:
Can we refer to the original name of the metric info `KafkaMetricDef.CPU_USAGE.name()` -- i.e. rather than creating this static variable for its name?

Reply:
The reason I do not do that is because this `Utils` class is in `cruise-control-core` module, to successfully import we need make `cruise-control-core` depend on `cruise-control`, which I think we do not want(since `cruise-control-core` should be a general library module).